docs/src/main/docs/health/01_health.adoc

```diff
@@ -0,0 +1,133 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Health Checks
+:description: Helidon health checks
+:keywords: helidon, health-checks, health, check
+
+== Health Check Support
+Health check support in Helidon SE provides:
+
+1. Endpoint `/health` - a configurable endpoint that exposes health checks in JSON
+    format specified by Microprofile Health Check specification
+2. Base set of health checks (additional dependency)
+
+Health Check Support in Helidon SE serves as a core for Helidon MP implementation of Microprofile Health Check specification.
+
+
+=== Prerequisites
+
+Declare the following dependency in your project:
+
+[source,xml,subs="verbatim,attributes"]
+.Health Check Dependency
+----
+    <dependency>
+        <groupId>io.helidon.health</groupId>
+        <artifactId>helidon-health</artifactId>
+    </dependency>
+----
+
+To enable built-in health checks, add the following dependency:
+
+[source,xml,subs="verbatim,attributes"]
+.Health Checks Dependency
+----
+    <dependency>
+        <groupId>io.helidon.health</groupId>
+        <artifactId>helidon-health-checks</artifactId>
+    </dependency>
+----
+
+
+=== Using Health Checks in Your Application
```

Comment:
See my earlier comment about using the sentence case for section headings. So this would become: "Using health checks in your application"

Also, I suggest "Adding..." instead of "Using..."